source/api/commands/should.md

```diff
@@ -166,9 +166,54 @@ cy
   })
 ```
 
-### Assert explicitly within `.should()`
+### Assert class name starts with `heading-`
+
+```html
+<div class="docs-header">
+  <div class="main-abc123 heading-xyz987">Introduction</div>
+</div>
+```
+
+```js
+cy.get('.docs-header')
+  .find('div')
+  // .should(cb) callback function will be retried
+  .should(($div) => {
+    expect($div).to.have.length(1)
+    
+    const className = $div[0].className
+
+    expect(className).to.match(/heading-/)
```

Comment:
If you want to assert that the className *starts* with 'heading-' as the title of this section says, this should be `/$heading-/` yes?

Reply:
yes, makes sense, I would need to rewrite either the assertion to split the names or change what I call this